beta-src/src/components/ui/WDInfoPanel.tsx

```diff
@@ -8,21 +8,31 @@ import { CountryTableData } from "../../interfaces/CountryTableData";
 import GameOverviewResponse from "../../state/interfaces/GameOverviewResponse";
 import useViewport from "../../hooks/useViewport";
 import getDevice from "../../utils/getDevice";
+import { toggleVoteStatus } from "../../state/game/game-api-slice";
+import { useAppDispatch } from "../../state/hooks";
 
 interface WDInfoPanelProps {
   countries: CountryTableData[];
+  gameID: GameOverviewResponse["gameID"];
   maxDelays: GameOverviewResponse["excusedMissedTurns"];
   userCountry: CountryTableData;
 }
 
 const WDInfoPanel: React.FC<WDInfoPanelProps> = function ({
   countries,
+  gameID,
   maxDelays,
   userCountry,
 }): React.ReactElement {
   const [voteState, setVoteState] = React.useState(userCountry.votes);
   const [viewport] = useViewport();
   const device = getDevice(viewport);
+  const dispatch = useAppDispatch();
+  const voteCommands = {
+    pause: "Pause",
+    draw: "Draw",
+    cancel: "Cancel",
+  };
```

Comment:
You shouldn't need this, voteKey is already the text of the vote. So you can just capitalize the first letter when dispatching.